backend/routes/upload_reservation_routes.py

```
@@ -183,18 +192,38 @@ def upload_file():
                     email_result = email_service.send_admin_checkin_notification(admin_email, checkin_data)
 
                     if email_result.get('status') == 'success':
-                        print(f"Admin notification sent successfully to {admin_email}")
+                        logger.info("Admin notification sent successfully", extra=safe_extra_fields({
+                            'admin_email': admin_email,
+                            'reservation_id': reservation_id,
+                            'client_name': f"{name} {surname}",
```

Ruff (0.13.1)

Undefined name `name`

(F821)